src/Containers/RightSideBar/Requirement.ts

```diff
@@ -0,0 +1,25 @@
+export interface RequirementProps {
```

Comment:
All interfaces should start with 'I' (interfaces that aren't technically props I believe).